ta/pkcs11/src/processing_asymm.c

```diff
@@ -642,6 +674,8 @@ enum pkcs11_rc step_asymm_operation(struct pkcs11_session *session,
 						    out_buf, &out_size);
 			output_data = true;
 			rc = tee2pkcs_error(res);
+			if (rc == PKCS11_CKR_ARGUMENTS_BAD)
+				rc = PKCS11_CKR_DATA_LEN_RANGE;
```

Comment:
This change seems to specific to RSA PKCS#1 OAEP.
Could you move to a specific commit, so we can better highlight why this post processing test was added?

I think your proposal is fine in that we are confident in the other params passed to `TEE_AsymmetricEncrypt()`/`TEE_AsymmetricDecrypt()`  are fine and only a bad input data size could make them return `TEE_ERROR_BAD_PARAMETERS`.

Reply:
OK. Splitted the next one too.